sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_helpers.py

```diff
@@ -123,3 +123,10 @@ def convert_datetime(date_time):
             except ValueError:
                 return datetime.datetime.strptime(date_time, "%Y-%m-%d %H:%M:%S")
     raise TypeError("Bad datetime type")
+
+
+def convert_order_by(order_by):
+    # type: (Optional[List[str]]) -> Optional[List[str]]
+    if order_by:
+        order_by = [order.replace("created_on", "createdDateTimeUtc") for order in order_by]
```

Comment:
would this work if the user passes any comb of upper case and lower case for `created_on` like i.e. `Created_on`?
not sure if this is a scenario, but asking in case it is important in Python

Reply:
Hmm this wouldn't be proper snake case so I wouldn't consider it a scenario. I think I'll let the service validate here, they do send back an error if the query param is not supported.